
Don't print pid of forked process on startup. This change was also made
upstream in 20f5b3886761a0ba963fee435bba90dd09bd5bd5.

  -- Chris Lamb <lamby@debian.org>  Thu, 14 Jan 2010 15:48:04 +0000

diff -urNad /tmp/bp-build/redis-1.2.0.orig/redis.c /tmp/bp-build/redis-1.2.0/redis.c
--- redis-1.2.0.orig/redis.c	2010-01-14 15:46:33.000000000 +0000
+++ redis-1.2.0/redis.c	2010-01-14 15:46:51.000000000 +0000
@@ -6215,7 +6215,6 @@
     FILE *fp;
 
     if (fork() != 0) exit(0); /* parent exits */
-    printf("New pid: %d\n", getpid());
     setsid(); /* create a new session */
 
     /* Every output goes to /dev/null. If Redis is daemonized but
